src/common/Network/CommonActorControl.h

```diff
@@ -311,12 +311,13 @@ enum ActorControlType : uint16_t
     RequestLandSignOwned = 0x452,
     RequestWardLandInfo = 0x453,
     RequestLandRelinquish = 0x454,
+    RequestExternalHousingInventory = 0x0458,
     RequestEstateRename = 0x45A,
     RequestEstateEditGreeting = 0x45B,
     RequestEstateGreeting = 0x45C, // sends FFXIVIpcHousingEstateGreeting in return
     RequestEstateEditGuestAccessSettings = 0x45D,
     RequestEstateTagSettings = 0x45F,
-
+    RequestInternalHousingInventory = 0x0461,
```

Comment:
Should probably just be called EstateInventory and LandInventory for consistency